sdk/src/account_meta.rs

```diff
@@ -0,0 +1,119 @@
+//! Types used
+
+use anchor_lang::{AnchorDeserialize, AnchorSerialize};
+use solana_program::pubkey::Pubkey;
+
+use crate::{
+    compressed_account::CompressedAccountWithMerkleContext,
+    merkle_context::{
+        pack_address_merkle_context, pack_merkle_context, AddressMerkleContext,
+        PackedAddressMerkleContext, PackedMerkleContext, RemainingAccounts,
+    },
+};
+
+#[derive(Debug, Clone, AnchorSerialize, AnchorDeserialize, PartialEq, Default)]
+pub struct LightAccountMeta {
+    /// Lamports.
+    pub lamports: Option<u64>,
+    /// Address of the account (the address can change).
+    pub address: Option<[u8; 32]>,
+    /// Data of the account.
+    pub data: Option<Vec<u8>>,
+    /// Merkle tree.
+    pub merkle_context: Option<PackedMerkleContext>,
+    /// Merkle tree root index.
+    pub merkle_tree_root_index: Option<u16>,
+    /// Output Merkle tree.
+    pub output_merkle_tree_index: Option<u8>,
```

Comment:
Here I think it should be optional, because we don't have an "output MT" when we're closing an account (provide input without expecting output).

As much as I agree with not having it optional in `new_mut` (already pushed), I would leave the Option here.